CMakeLists.txt

```diff
@@ -69,8 +69,8 @@ option(FIREBASE_INCLUDE_REMOTE_CONFIG
 option(FIREBASE_INCLUDE_STORAGE
        "Include the Cloud Storage for Firebase library."
        ${FIREBASE_INCLUDE_LIBRARY_DEFAULT})
-option(FIREBASE_INCLUDE_VERTEXAI
-       "Include the Vertex AI in Firebase library."
+option(FIREBASE_INCLUDE_FIREBASEAI
```

Comment:
since we choose FIREBASE_INCLUDE_DYNAMIC_LINKS and  FIREBASE_INCLUDE_REMOTE_CONFIG, which makes me feel this should be FIREBASE_INCLUDE_FIREBASE_AI

Reply:
It is also based on the directory structure. Since we decided on "firebaseai" without the underscore, I think it makes sense to not have it here either, to be consistent.

Reply:
See some other cases which makes more sense to get FIREBASEAI all together while using upper cases. So let's keep it as FIREBASEAI